Tools/cases_generator/generators_common.py

```diff
@@ -178,58 +242,274 @@ def decref_inputs(
                     self.out.emit(f"PyStackRef_XCLOSE({var.name});\n")
             else:
                 self.out.emit(f"PyStackRef_CLOSE({var.name});\n")
+        for input in storage.inputs:
+            input.defined = False
+        return True
+
+    def kill_inputs(
+        self,
+        tkn: Token,
+        tkn_iter: TokenIterator,
+        uop: Uop,
+        storage: Storage,
+        inst: Instruction | None,
+    ) -> bool:
+        next(tkn_iter)
+        next(tkn_iter)
+        next(tkn_iter)
+        for var in storage.inputs:
+            var.defined = False
+        return True
+
+    def kill(
+        self,
+        tkn: Token,
+        tkn_iter: TokenIterator,
+        uop: Uop,
+        storage: Storage,
+        inst: Instruction | None,
+    ) -> bool:
+        next(tkn_iter)
+        name_tkn = next(tkn_iter)
+        name = name_tkn.text
+        next(tkn_iter)
+        next(tkn_iter)
+        for var in storage.inputs:
+            if var.name == name:
+                var.defined = False
+                break
+        else:
+            raise analysis_error(f"'{name} is not a live input-only variable", name_tkn)
```

Comment:
```suggestion
            raise analysis_error(f"{name!r} is not a live input-only variable", name_tkn)
```

Reply:
For this one, you should just add a missing quote `'{name}'` instead of `'{name}`